vm/src/utils.rs

```diff
@@ -127,3 +127,40 @@ impl ToCString for PyStr {
         std::ffi::CString::new(self.as_ref()).map_err(|err| err.into_pyexception(vm))
     }
 }
+
+pub(crate) fn repr_helper<'a, I>(
+    class_name: Option<&str>,
+    brackets: [char; 2],
```

Comment:
This variable is always used only for each char, but never as array. If we change this to `prefix: &str, suffix: &str`, we can handle more cases. For example, `deque` is now formatted again using maxlen. But if we just pass suffix string, we can pass right bracket and maxlen at same time.